problems/http.go

```diff
@@ -0,0 +1,49 @@
+package problems
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/SKF/go-utility/v2/log"
+)
+
+// Generic, returns a generic HTTP-based Problem from a HTTP status code.
+func Generic(status int) Problem {
+	return BasicProblem{
+		Title:  http.StatusText(status),
+		Status: status,
+	}
+}
+
+// WriteResponse, converts the error into a Problem and writes the contents into w.
+// The Problem will be decorated with request information and logged if possible.
+func WriteResponse(ctx context.Context, err error, w http.ResponseWriter, r *http.Request) {
+	problem := FromError(err)
+
+	if decoratableProblem, ok := problem.(RequestDecoratableProblem); ok {
```

Comment:
Verify that this works for validationProblem

Reply:
This works as intdended